guides/release/upgrading/index.md

```diff
@@ -1 +1,81 @@
+7. Create a new component in your app, and experiment!
+8. Try refactoring some existing code to use a new feature. Check out the [cheat sheet](../cheat-sheet/) and [Editions deep dive](../editions/) for some pointers.
+8. Schedule a show-and-tell meeting with your coworkers to fill them in about what you learned 
+9. Review the refactoring checklist below to create a plan for handling existing code
+
+If you need help along the way, visit [the Ember Community chat and forums](https://emberjs.com/community/).
+
+### Deprecations that matter for Octane
+
+- [`jquery-apis`](https://deprecations.emberjs.com/v3.x/#toc_jquery-apis) - By default, Octane does not include jQuery. Continuing to use jQuery in your app will not conflict with Octane features, however you should follow the deprecation instructions to keep using it.
+
+<!-- TODO -->
+
+### Optional features in Octane
+
+A fully-Octane app has the following configuration in `config/optional-features.json`:
+
+```json
+{
+  "application-template-wrapper": false,
+  "jquery-integration": false,
+  "template-only-glimmer-components": true
+}
+```
+
+Use the command `ember feature:list` in your console to learn what each option does.
+
+### Refactoring checklist
+
+For many of the optional features, the thing they affect the most is what you see in newly created files, not your existing code.
+That's a good place to start, but eventually you may want to refactor existing code so that your app follows one main programming model, not a mixture of Octane and Classic.
+Following a refactoring plan will help with onboarding new developers, and minimize flipping back and forth between different versions of the Ember Guides.
+
+There's no one-size-fits-all strategy, but here is a checklist you can adapt, once you're familiar with what Octane has to offer:
+
+1. Convert curly bracket components (`{{my-component}}`) to Angle Brackets (`<MyComponent />`). They are a normal, out-of-the-box feature of Ember since [3.4](https://blog.emberjs.com/2018/10/07/ember-3-4-released.html) that does not change a component's behavior. Follow the [Angle Bracket Syntax guide](../../reference/syntax-conversion-guide) for examples.
+2. Use Named Arguments and `this` in your templates. This also does not change component behavior.
+3. Refactor some small, basic components to use Native Classes. Good components to refactor first are those that do not rely on two-way binding, computed properties, or observers. These components will serve as examples that your coworkers can refer back to.
+5. Refactor components that have computed properties to use Native Classes and `@tracked`
+6. Refactor remaining components, such as those with two-way-binding and Observers.
+
+At any point in this process, you could refactor your Routes, Controllers, Services, and Models to use Native Classes too. However, the strategy above is centered on Components because they are the biggest shift in mental model and the bulk of the files in an app.
```

Comment:
```suggestion
At any point in this process, you could refactor your Routes, Controllers, Services, and Models to use Native Classes too. However, the strategy above is centered on Components because they are the biggest shift in the mental model of Octane and affect the bulk of the files in an app.
```